### ISSUE TITLE
ReferenceError: scan is not defined

### ISSUE THREAD

**Issue by reality-post-productions** [yetzt/node-ftpdump#2]
Raised error on ftpdump.js, line 87:
```txt
ReferenceError: scan is not defined
    at /Users/reality/Desktop/node-ftpdump/ftpdump.js:87:4
    ...
```
Try changing: 

```js
// this line
scan(fn);

// ===> into that
self.scan(fn);
```

Also some extra thoughts:

1. The output path for the downloaded content is not very clear. It generates the whole directory tree until the current dir I want. It would be useful to tweak this behavior
2. It's not very intuitive to use the `debug` module for progresses. Instead it would be nice to use a spinner ([`ora`](https://www.npmjs.com/package/ora) or [`clui`](https://www.npmjs.com/package/clui)  are nice choices), but I will try for a pull request asap.

**Comment by yetzt** [comment]
thanks for the bug report, i'll look into it.

**Comment by yetzt** [comment]
fixed in 85817bb9
